### DISCUSSION TITLE
add plugin to project

### DISCUSSION THREAD

**Discussion by ilksenbozok** [serenity-is/Serenity#6767] [Help]
Hello,
I am following the How To: Use a Third Party Plugin With Serenity guide. While creating the file "BSMultiSelectEditor.ts" I get the following error : 
![image](https://github.com/serenity-is/Serenity/assets/27086236/80143343-6d48-4813-b925-301165c6f64a)

I searched all resources, opened a new project and tried again and again, the problem was not solved.
I also uploaded the required css and js files at the beginning

**Comment by edwardch** [comment]
Check the changelogs for 6.6.6 (https://github.com/serenity-is/Serenity/blob/master/CHANGELOG.md#660-2023-03-14)

  - [BREAKING CHANGE] There is no longer a globally defined Q.Lookup interface while using ES modules. If you referenced it in some modular code, you should `import { Lookup } from "serenity-is/corelib/q"` and replace `Q.Lookup` with `Lookup` in your project.

Can you try that first?
